pom.xml

```diff
@@ -285,6 +303,7 @@
 					<artifactId>tycho-compiler-plugin</artifactId>
 					<version>${tycho-version}</version>
 					<configuration>
+						<release>${java.version}</release>
```

Comment:
I don't think it's a good idea. The "release" value is supposed to be dynamically deduced from the Bundle-RequiredExecutionEnvironment or osgi.ee requirement. Hardcoding makes that having different BREEs in the project is not well supported anymore (imagine 1 bundle wants JavaSE-1.8 and another JavaSE-17), and makes migration to different/newer JRE harder because this pom would need to be modified.
I'm going to try removing it, and the following setting as well; since the workaround mentioned is supposed to be available in Tycho 2.5.